packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -919,6 +922,8 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 				}
 
 				this.orderProperties = newOrderProperties; // enforcing immutability :)
+				this.isMultiSorting = this.orderProperties.length > 1;
+				this.cdRef.markForCheck(); // Force the change https://angular.io/api/core/ChangeDetectorRef#usage-notes 
```

Comment:
Why do we need `this.cdRef.markForCheck()` ?
It seems to work without this. Could you check ?